packages/react-scripts/package.json

```diff
@@ -21,37 +21,38 @@
     "react-scripts": "./bin/react-scripts.js"
   },
   "dependencies": {
-    "autoprefixer": "7.1.1",
-    "babel-core": "6.25.0",
-    "babel-eslint": "7.2.3",
+    "autoprefixer": "^7.1.1",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "8.1.1",
     "babel-jest": "20.0.3",
-    "babel-loader": "7.1.1",
+    "babel-loader": "^7.1.4",
     "babel-preset-react-app": "^3.0.1",
-    "babel-runtime": "6.23.0",
+    "babel-runtime": "6.26.0",
     "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "chalk": "1.1.3",
+    "chalk": "^1.1.3",
     "css-loader": "0.28.4",
-    "dotenv": "4.0.0",
-    "eslint": "4.1.1",
-    "eslint-config-react-app": "^1.0.5",
+    "dotenv": "^4.0.0",
+    "eslint": "^4.14.0",
+    "eslint-config-react-app": "^2.1.0",
     "eslint-loader": "1.9.0",
-    "eslint-plugin-flowtype": "2.34.0",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-jsx-a11y": "5.1.1",
-    "eslint-plugin-react": "7.1.0",
+    "eslint-plugin-flowtype": "^2.46.1",
+    "eslint-plugin-import": "^2.9.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-react": "^7.7.0",
     "extract-text-webpack-plugin": "3.0.0",
     "file-loader": "0.11.2",
-    "fs-extra": "3.0.1",
-    "node-sass": "^4.5.3",
-    "sass-loader": "^6.0.6",
+    "fs-extra": "^3.0.1",
     "html-webpack-plugin": "2.29.0",
-    "jest": "20.0.4",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
This dependency is used by Jest for the `moduleNameMapper` key for `css` and `scss` files.